### PR TITLE
Python 3 compatible 

### PR DESCRIPTION
**Pull request by 5up3rD4n1**
There are some things I'm not sure about, try it out and let me know if it needs some change!

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -8,10 +8,13 @@
                                             'chdkptp'))
 CHDKPTP_PATCH = os.path.abspath(os.path.join('.', 'chdkptp_module.diff'))
 
+print(CHDKPTP_PATH)
+print(CHDKPTP_PATCH)
+
```
Comment (jbaiter) [inline setup.py]: Could you remove those?

File: README.rst [hunk 2]
```diff
@@ -2,6 +2,76 @@
 chdkptp.py
 ==========
 
+This repo is a fork from  `chdkptp.py <https://github.com/jbaiter/chdkptp.py>`_
```
Comment (jbaiter) [inline README.rst]: Could you fix the repository URLs?

File: chdkptp/lua.py [hunk 3]
```diff
@@ -25,7 +24,7 @@ class LuaContext(object):
         inside of `pcall` and raises proper Exceptions.
     """
     def _raise_exception(self, errval):
-        if isinstance(errval, (basestring, numbers.Number)):
+        if isinstance(errval, (str, numbers.Number)):
```
Comment (jbaiter) [inline chdkptp/lua.py]: Have you checked if this works with Python 2? I'm not sure if lupa maps Lua strings to the `str` or `unicode` type. If it's the latter, we might need a workaround to maintain support for Python 2.x.

**Review by jbaiter — COMMENTED** [review]
Looks good to me, apart from a few minor things listed in the review comments. It would be great if you could confirm that your code works with Python 2.7, I'd hate to lose backwards compatibility :-)

Did you check if the code in `device.py` still works with your new chdkptp version?

File: README.rst (hunk 2 above)
Comment (5up3rD4n1) [reply README.rst]: Sure, but which should be the correct one?

File: chdkptp/lua.py (hunk 3 above)
Comment (5up3rD4n1) [reply chdkptp/lua.py]: Yeah, I'm not sure about that, I read somewhere just to replace the basestring to str, I'm going to do a deeper research about it, just to be sure.

**Comment by 5up3rD4n1** [comment]
I found that `str` and `basestring` can be changed with `six.string_types` which is also compatible with python2 and python3. I also changed the readme so you can add some installation information, please fell free to change the release tar.gz file and all that when you have your release.  I also compile it and run it with python2.7 and seems to work fine. :+1:

**Pull request closed without merging**